shared/common-adapters/avatar.d.ts

```diff
@@ -16,7 +16,7 @@ export type Props = {
   isTeam?: boolean
   loadingColor?: string
   onClick?: (e?: React.SyntheticEvent) => void
-  onEditAvatarClick?: (e?: React.SyntheticEvent) => void | null
+  onEditAvatarClick?: ((e?: React.SyntheticEvent) => void )| null
```

Comment:
we'll take a further pass to do the whole undefined instead of null thing. i realized the change was better halfway through and only did it to reconcile a cluster of issues. we'll clean it up as we move forward